src/WebJobs.Extensions.DocumentDB/DocumentDBAttribute.cs

```diff
@@ -64,8 +66,24 @@ public DocumentDBAttribute(string databaseName, string collectionName)
 
         /// <summary>
         /// Optional. The Id of the document to retrieve from the collection.
-        /// May include binding parameters
+        /// May include binding parameters.
         /// </summary>
         public string Id { get; set; }
+
+        /// <summary>
+        /// Optional.
+        /// When specified on an output binding and <see cref="CreateIfNotExists"/> is true, defines the partition key 
+        /// path for the created collection.
+        /// When specified on an input binding, specifies the partition key value for the lookup.
+        /// May include binding parameters.
+        /// </summary>
+        public string PartitionKey { get; set; }
+
+        /// <summary>
+        /// Optional.
+        /// When specified on an output binding and <see cref="CreateIfNotExists"/> is true, defines the throughput of the created
+        /// collection.
+        /// </summary>
+        public int CollectionThroughput { get; set; }
```

Comment:
I've pinged the DocDB PM to see if he has input. 'Offer' sounded too generic to me and I wasn't sure if that property was used elsewhere. I'll do whatever he suggests :-)

---
In reply to: [59797287](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/63#discussion_r59797287) [](ancestors = 59797287)